roles/pulp_common/tasks/install.yml

```diff
@@ -2,18 +2,18 @@
 - name: General system changes before installation
   block:
 
-    - name: Enable Python 3.8 module stream
-      command: dnf -y module enable python38
-      register: python38_module_enable
```

Comment:
It would be great if this was somehow configurable, instead of hard-coded.  It would make future changes to python interpreters a simple setting in the config, where the rest of the changes are happening, instead of having to change this specific task in a separate file.

Reply:
For 3.18 we got a request from Katello to keep building for EL7, with this we will keep python 3.8 for EL7 only. +1 for @pcreech on finding a way to get it configurable.

Reply:
Done